src/main/frontend/src/app/ltistorage.js

```diff
@@ -0,0 +1,488 @@
+//  This code originates from 1EdTech's LTI Postmessage Example Library.
```

Comment:
Just out of curiosity, why does this code need to exist twice?

Reply:
the react app inside needs access inside its own `/src` file.

Reply:
Ah.  That's lamentable but we can try to think through a fix to that in the future sometime....